streamz-converter/src/main/scala/streamz/converter/Converter.scala

```diff
@@ -16,102 +16,131 @@
 
 package streamz.converter
 
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.{ Failure, Success }
+
 import akka.stream._
 import akka.stream.scaladsl.{ Flow => AkkaFlow, Sink => AkkaSink, Source => AkkaSource, _ }
 import akka.{ Done, NotUsed }
+import cats.MonadError
 import cats.effect._
 import cats.effect.concurrent.Deferred
 import cats.effect.implicits._
 import cats.implicits._
-import fs2._
-
-import scala.concurrent.{ ExecutionContext, Future }
-import scala.util.{ Failure, Success }
+import fs2.{ Pipe, _ }
```

Comment:
```suggestion
import fs2._
```

Normally I would avoid wildcard importing packages that aren't `implicits`/`syntax` and designed for it, but since it was here before I won't ask you to change it (though if you want to I'd take it)